src/interfaces/email/email.interface.ts

```diff
@@ -1,7 +1,4 @@
 /* eslint-disable camelcase */
```

Comment:
I think this rule ignore command can be deleted now since all is respecting camelcase

Reply:
done, thank you !